style.css

```diff
@@ -0,0 +1,901 @@
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: "Lato", sans-serif;
+}
+
+ul {
+  list-style: none;
+}
+
+/* ========== mobile-veiw =============== */
+
+/* =========== first section Home page ======== */
+
+.main-section {
+  background-image: url(img/mobile_intro_bg.png);
+  background-size: contain;
+  background-position: center;
+  background-repeat: no-repeat;
+  display: flex;
+  flex-direction: column;
+  padding: 1rem;
+  margin-bottom: 3rem;
+}
+
+.main-section-article {
+  margin-top: 4rem;
+}
+
+article h2 {
+  font-size: x-large;
+  font-weight: 900;
+  text-transform: uppercase;
+}
+
+article h1,
+article h2 {
+  color: #ec5342;
+}
+
+.main-section-article h1,
+.main-section-article h2 {
+  width: 90%;
+  text-align: left;
+  padding-bottom: 1rem;
+  margin: auto 1rem;
+}
+
+.main-section-article h2 {
+  margin-bottom: 2rem;
+}
+
+.article-1 {
+  color: #272a31;
+  padding-top: 1rem;
+  padding-bottom: 1rem;
+  margin-left: 2rem;
+  background-color: white;
+  width: 80%;
+  height: 38vh;
+  text-align: left;
+}
+
+#hamburger {
+  position: fixed;
+  z-index: 999;
+  cursor: pointer;
+  color: #ec5342;
+  font-size: 40px;
+  margin-top: 1rem;
+  width: 100%;
+}
+
+.menu-container {
+  display: flex;
+  flex-direction: row-reverse;
+  justify-content: space-between;
+  position: fixed;
+  z-index: 999;
+  width: 100%;
+  background-color: white;
+  margin-left: -1rem;
+  margin-top: -1rem;
+}
```

Comment:
- [ ] Please change the navbar to follow the design guidelines. Refer to this design:
![image](https://user-images.githubusercontent.com/92215878/192031722-28481fe9-000b-43ec-8e74-8ba89aa228c0.png)